acceptance/bundle/repos/create_with_provider/out.test.toml

```diff
@@ -0,0 +1,5 @@
+Local = true
+Cloud = false
+
+[EnvMatrix]
+  DATABRICKS_CLI_DEPLOYMENT = ["terraform", "direct-exp"]
```

Comment:
This is not relevant for these tests. Moving them outside of `bundle` will address this.

Reply:
These config are serialized for tests outside `bundle` too.